### DISCUSSION TITLE
Preserve wide ranges for a dependency

### DISCUSSION THREAD

**Discussion by parthea** [renovatebot/renovate#17594] [Request Help]
In PR https://github.com/googleapis/python-video-live-stream/pull/104/files, renovate bot is proposing to restrict the range for dependency `google-api-core` from `"google-api-core[grpc] >= 1.32.0, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*",` to `"google-api-core[grpc] >=2.10, <3.0.0dev, !=2.0, !=2.1, !=2.2, !=2.3, !=2.4, !=2.5, !=2.6, !=2.7",`  however we want to keep the original range as it is wider and still supports `google-api-core==2.10.0` which was published yesterday.

Please could you help suggest a configuration change to prevent restricting the range for dependencies ?

**Comment by rarkins** [comment]
Which rangeStrategy are you using currently?

**Comment by parthea** [comment]
I don't have a rangeStrategy defined. It looks like the default is `replace` : `Replace the range with a newer one if the new version falls outside it, and update nothing otherwise`

**Reply by rarkins** [reply]
The confusing thing is that `replace` shouldn't make any changes at all if the new version is compatible with the existing range. And it does look like `2.10.0` falls within the existing range, right?
